### PR TITLE
docs: example on how to include a custom function in the data model

### PR DESCRIPTION
**Pull request by mahakporwal02**
**Description**
Added example on how to include custom function in data model.
https://github.com/mahakporwal02/modelina/tree/issue-413/examples/include-custom-function
Updated documentation.
https://github.com/mahakporwal02/modelina/blob/issue-413/examples/include-custom-function/README.md
https://github.com/mahakporwal02/modelina/blob/issue-413/examples/README.md
https://github.com/mahakporwal02/modelina/blob/issue-413/docs/advanced.md

**Related issue(s)**
#413

### CODE REVIEW

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
Got a few comments, otherwise looks good 👍

File: docs/advanced.md [hunk 1]
```diff
@@ -41,7 +41,10 @@ Supported by:
 Check out this [example out for a live demonstration](../examples/generate-to-files).
 
 ## Include a custom function in the data model
-TODO 
+Sometime you want to include some custom functionality into the generated models, and here is how.
+This can be done through a custom preset using the hook additionalContent.
```
[the remark this answers is not in the document]
Comment (mahakporwal02) [reply docs/advanced.md]: Definitely.

**Comment by mahakporwal02** [comment]
@jonaslagoni `examples/generate-to-files/index.spec.ts` test passes in my local repository. I don't know why it is failing here.

File: examples/include-custom-function/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# TODO: Custom Function
```
Comment (jonaslagoni) [inline examples/include-custom-function/README.md:1]:
```suggestion
# Custom Function
```

File: examples/README.md [hunk 3]
```diff
@@ -25,4 +25,5 @@ This directory contains a series of self-contained examples that you can use as
 - [javascript-use-esm](./javascript-use-esm) - A basic example that generate the models to use ESM module system.
 - [javascript-use-cjs](./javascript-use-cjs) - A basic example that generate the models to use CJS module system.
 - [generate-java-models](./generate-java-models) - A basic example to generate Java data models.
-- [generate-go-models](./generate-go-models) - A basic example to generate Go data models
\ No newline at end of file
+- [generate-go-models](./generate-go-models) - A basic example to generate Go data models
+- [custom-function](./custom-logging) - A basic example where a custom function is included.
```
Comment (jonaslagoni) [inline examples/README.md:29]:
```suggestion
- [include-custom-function](./include-custom-function) - A basic example where a custom function is included.
```

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
Just to more changes 🙂 

Regarding the tests not passing, it should be resolved now.

**Review by jonaslagoni — APPROVED** [review]
LGTM 👍 Awesome 🎉

**Pull request merged**